### PR TITLE
chore(Replies): Clean up the UI for replies to make them more clear.

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖

WIP

### CODE REVIEW

**Comment by InfamousVague** [comment]
image

**Comment by InfamousVague** [comment]
image
User replies are now split into groups to make things more clear as well

File: kit/src/components/message_reply/mod.rs [hunk 1]
```diff
@@ -125,6 +125,14 @@ pub fn MessageReply<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
             (cx.props.user_image.is_some() && !remote_message).then(|| rsx! (
                 cx.props.user_image.as_ref()
             )),
+            div {
+                class: "connector",
+                if cx.props.remote.unwrap_or_default() {
+                    "┌"
```
Comment (lgmarchi) [inline kit/src/components/message_reply/mod.rs:131]: just small suggestion here. Add these connectors as const like. 

const RIGHT_REPLY_CONNECTOR: &str = "┌"; 

const LEFT_REPLY_CONNECTOR: &str = "┐";

**Review by lgmarchi — APPROVED** [review]
PR Approved. Just left a small suggestion.

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Ready to merge once CI is done

File: kit/src/components/message_reply/mod.rs (hunk 1 above)
Comment (InfamousVague) [reply kit/src/components/message_reply/mod.rs:131]: I like this idea, I'll add this to a future un-related PR.

**Pull request merged**